pkg/controller/storagecluster/reconcile.go

```diff
@@ -1152,3 +1156,129 @@ func getMonCount(nodeCount int) int {
 
 	return count
 }
+
+//ensureJobTemplates returns spec of a job template
```

Comment:
I see it only returns `error`??

Reply:
That is the ensureFunc template, so yes that's all i can return.

Reply:
@xenolinux you need to wrap this around a user-friendly method